test/shell_test.exp

```diff
@@ -423,6 +423,8 @@ njs_run {"-c" "console.log(process.pid)"} "\\d+"
 
 njs_run {"-c" "console.log(process.ppid)"} "\\d+"
 
+njs_run {"-c" "console.log(process.kill(process.pid, 0))"} "true"
```

Comment:
Should I comment this line out?

I guess it didn't run with my local `make test`, probably because I didn't have QuickJS installed.